lib/orbf/rules_engine/data/contract.rb

```diff
@@ -1,17 +1,46 @@
 module Orbf
   module RulesEngine
+    class ContractValidationException < StandardError
+      def initialize(missing_attrs, org_unit)
+        org_unit_missing = missing_attrs.include?("org_unit")
```

Comment:
If we make this `org_unit ||= {}` we don't have to do the `!org_unit_missing` check, since the `org_unit.fetch("name") {"missing_org_unit_name"}` will return `missing_org_unit_name`.

And then we can make the message always be `"The following attributes are missing for this contract: #{missing_attrs}" (orgunit_name - #{org_unit_name || 'missing'},orgunit_ id - #{org_unit_id || 'missing')

So we lose the ternary (mostly because I don't like ternaries, but also because it makes the code a little bit easier to read?)